open-hackathon-client/src/client/templates/create_event.html

```diff
@@ -0,0 +1,346 @@
+{% extends "base/manage.html" %}
+
+{% block style %}
+    <link href="/static/css/daterangepicker-bs3.css" rel="stylesheet">
+    <link href="/static/css/bootstrapValidator.css" rel="stylesheet">
+    <link href="/static/css/bootstrap-markdown.min.css" rel="stylesheet">
+    <style>
+        body {
+            background: none;
+        }
+
+        body:after {
+            content: initial;
+        }
+    </style>
+{% endblock %}
+
+{% block content %}
+    <div id="content" class="container clearfix">
+        <div class="col-lg-12">
+            <div class="box">
+                <div class="box-header">
+                    <h2><i class="fa fa-edit"></i>创办黑客松
+                    </h2>
+                </div>
+                <div class="box-content">
+                    <div class="step-content">
+                        <div class="step-pane active" id="step1">
+                            <form class="form-horizontal" id="stepform1" method="post"
+                                  data-bv-feedbackicons-valid="glyphicon glyphicon-ok"
+                                  data-bv-feedbackicons-invalid="glyphicon glyphicon-remove"
+                                  data-bv-feedbackicons-validating="glyphicon glyphicon-refresh">
+                                <fieldset>
+                                    <legend>
+                                        1 创建黑客松基本信息
+                                    </legend>
+                                    <div class="">
+                                        <div class="form-group">
+                                            <label class="required col-sm-2 control-label">黑客松名称</label>
+
+                                            <div class="col-sm-10">
+                                                <input type="text" class="form-control" name="name" id="name"
+                                                       placeholder="请使用英文字母、数字和'_'（下划线）'-'(中划线)"
```

Comment:
Please add a more constraint that the `name` of hackathon *cannot* contain uppercase charactors